x-pack/plugins/secops/public/components/page/network/network_top_n_flow_table/columns.tsx

```diff
@@ -156,13 +147,14 @@ export const getNetworkTopNFlowColumns = (
     },
   },
   {
+    field: 'node.network.direction',
     name: i18n.DIRECTION,
     truncateText: false,
     hideForMobile: false,
-    render: ({ node }) =>
-      isEmpty(get('network.direction', node))
+    render: (directions: NetworkDirectionEcs[]) =>
+      isEmpty(directions)
         ? getEmptyTagValue()
-        : get('network.direction', node).map((direction: NetworkDirectionEcs) => (
+        : directions.map((direction: NetworkDirectionEcs, index: number) => (
```

Comment:
Optional: For typescript, when you do a `.map` you can leave off the types within the `.map` because it is smart enough to know what they are.

So you can do:

```ts
directions.map((direction, index) => (
```

And that will cut down on semantic noise.